devicetypes/vision-stevenchen/arrival-sensor-ha.src/arrival-sensor-ha.groovy

```diff
@@ -0,0 +1,197 @@
+import groovy.json.JsonOutput
```

Comment:
@Steven5517 I think You still need to revert Your changes in **devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy**  instead of creating a new file in:
**vision-stevenchen/arrival-sensor-ha.src/arrival-sensor-ha.groovy**

Reply:
OK! I will revert my changes in devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy.